flexget/plugins/plugin_transmission.py

```diff
@@ -298,6 +297,13 @@ def on_task_output(self, task, config):
         if task.accepted:
             self.add_to_transmission(self.client, task, config)
 
+    def _decode_if_bytes(self, data):
+        """Transmission doesn't like encoded byte strings (Py3) so decode"""
+        if type(data) == bytes:
```

Comment:
Basically i didn't have any other data to go on and i know it's an issue with this specific type in Py3. I didn't want to introduce other errors if things should change in the future with bytes.

I used isinstance later as i need to check for Py3 without checking for a Python version and it doesn't matter if it's descendants match.

If i'm wrong in my thinking just let me know and i'll change it!